gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -264,6 +264,23 @@ public void open_throwsExceptionWhenHadoopPathNull() {
     assertThat(exception).hasMessageThat().startsWith("hadoopPath must not be null");
   }
 
+  @Test
+  public void open_storageStatistics() throws IOException {
+    GoogleHadoopFileSystem myGhfs = createInMemoryGoogleHadoopFileSystem();
+
+    Path testRoot = new Path("/directory1/");
+    myGhfs.mkdirs(testRoot);
+
+    FSDataOutputStream fout = myGhfs.create(new Path("/directory1/file1"));
+
+    fout.writeBytes("data");
+    fout.close();
+    myGhfs.open(new Path("/directory1/file1"));
+    GhfsStorageStatistics storage_stats = myGhfs.getStorageStatistics();
+    assertThat(storage_stats.isTracked("op_open")).isTrue();
```

Comment:
Are there any other metrics we want assert here ?

Reply:
This method is to validate only open related statistics. Other metrics are validated in different methods